web/src/stream_settings_ui.js

```diff
@@ -811,14 +811,16 @@ export function launch(section, left_side_tab, right_side_tab) {
             },
         });
         change_state(section, left_side_tab, right_side_tab);
+        setTimeout(() => {
+            if (!stream_settings_components.get_active_data().id) {
+                if (section === "new") {
+                    $("#create_stream_name").trigger("focus");
+                } else {
+                    $("#search_stream_name").trigger("focus");
+                }
+            }
+        }, 100);
```

Comment:
Do we need a nonzero timeout here, or just 0?

Reply:
Yes, 0 should work.

Reply:
Fixing it up!